modules/microsoft.cdn/azure-frontdoor-cdn-profile/1.0/modules/diagnosticsettings.bicep

```diff
@@ -0,0 +1,62 @@
+@sys.description('Name of CDN Profile. For chaining, use output from parent module')
+param cdnProfileName string
+
+@sys.description('Event Hub Name')
+param eventHubName string
+
+@sys.description('Event Hub AuthId')
+param eventHubAuthId string
+
+resource cdn 'Microsoft.Cdn/profiles@2021-06-01' existing = {
+  name: cdnProfileName
+  scope: resourceGroup()
+}
+
+var diagnosticSettings = [
```

Comment:
move this above the existing resource declaration

Reply:
Fixed